pkgs/tools/audio/mpdris2/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, autoconf, automake, intltool
+, python, wrapPython, mpd, pygtk, dbus, pynotify
+}:
+
+stdenv.mkDerivation rec {
+  name = "mpDris2";
+  version = "0.6";
+
+  src = fetchurl {
+    url = "https://github.com/eonpatapon/${name}/archive/${version}.tar.gz";
+    sha256 = "0zdmamj2ldhr6y3s464w8y2x3yizda784jnlrg3j3myfabssisvz";
+  };
+
+  buildInputs = [ autoconf automake intltool ];
+  propagatedBuildInputs = [ python wrapPython ];
+  pythonPath = [ mpd pygtk dbus pynotify ];
+
+  preConfigure = "autoreconf -vfi";
+  postInstall = "wrapPythonPrograms";
+
+  meta = with stdenv.lib; {
+    description = "mpDris2 provides MPRIS 2 support to mpd.";
```

Comment:
Descriptions should not end with a period and contain the package name. See http://nixos.org/nixpkgs/manual/#idm140737319009616